tasks/libs/pipeline_tools.py

```diff
@@ -89,7 +89,8 @@ def trigger_agent_pipeline(
     all_builds=False,
     kitchen_tests=False,
     e2e_tests=False,
-    rc_k8s_deployments=False,
+    rc_build=False,
+    rc_k8s_deployments=True,
```

Comment:
AFAICS it doesn't change the behavior to change the default value here since the only callsite is providing the argument explicitly, but is the change required then?

Reply:
That's actually my mistake from the previous attempt when I thought about changing this flag to 'skip-rc-k8s-deployments', I missed reverting this.